### ISSUE TITLE
Update ZeppelinHub integration instructions, based on Zeppelin version

### ISSUE THREAD

**Issue by bzz** [NFLabs/zeppelinhub-documentation#15]
Current instruction under http://help.zeppelinhub.com/zeppelin_integration/ assume user with Zeppelin version at least `0.6.0-SNAPSHOT`, but before [#880](https://github.com/apache/incubator-zeppelin/pull/880) (wich is merged now).

We should update that page to include instructions at least for:
 - Zeppelin 0.5.6 (latest release, as of now)
 - 0.6.0 (next release, coming soon)
 - current master (in case it's different from 2 above)

**Comment by AhyoungRyu** [comment]
@bzz Good catch. I will update `zeppelin_integration` page for those info then :)
